src/jazayeri_lab_to_nwb/watters/watters_convert_session.py

```diff
@@ -110,11 +111,13 @@ def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str,
 
     # Add datetime to conversion
     metadata = processed_converter.get_metadata()  # use processed b/c it has everything
-    try:
-        date = datetime.datetime.strptime(data_dir_path.name, "%Y-%m-%d").replace(tzinfo=ZoneInfo("US/Eastern"))
-    except:
-        date = datetime.datetime(year=2022, month=6, day=1, tzinfo=ZoneInfo("US/Eastern"))
-    metadata["NWBFile"]["session_start_time"] = date
+    if "session_start_time" not in metadata["NWBFile"]:
+        try:
+            date = datetime.datetime.strptime(data_dir_path.name, "%Y-%m-%d").replace(tzinfo=ZoneInfo("US/Eastern"))
+        except:
+            print("Session start time not auto-detected. Setting to 2022-06-01...")
+            date = datetime.datetime(year=2022, month=6, day=1, tzinfo=ZoneInfo("US/Eastern"))
```

Comment:
Remind me of the reason for setting a default here instead of erroring and telling the user to set it manually?

Reply:
there's no good reason lol, throwing an error is a better solution, will fix